### DISCUSSION TITLE
Quenstion about three functions of the internal pytest API

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest#11262] [Q&A]
Hello

The API documentation isn't very clear and I have a question about these functions.
What are their purpose ?

```
pytest_configure(config)

config.addinivalue_line(*args)

parser.getgroup("group")
```

**Comment by The-Compiler** [comment]
I don't think I can say much more than what the documentation already says about those to be honest. Can you add a bit more context about what you're confused about?
